examples/alternate-receive/alternate_receive.md

```diff
@@ -0,0 +1,7 @@
+# Alternate receive
+
+The alternate receive action can be used to receive one of multiple values corresponding to multiple send actions. It operates by waiting until it encounters a non-error message, a panic termination status on a closed channel, or the closure of all channels. Alternate receive action sets the first non-error value it encounters as the result.
```

Comment:
We can also mention that it returns the last received error if all channels return errors.